src/main/java/com/netgrif/application/engine/elastic/service/ElasticCaseService.java

```diff
@@ -93,16 +93,24 @@ public Map<String, Float> fullTextFields() {
     @Override
     public void remove(String caseId) {
         executors.execute(caseId, () -> {
-            repository.deleteAllByStringId(caseId);
-            log.info("[" + caseId + "]: Case \"" + caseId + "\" deleted");
+            try {
```

Comment:
It would be probably better to place the try-catch inside the Executor.execute method so that it is only on one place.

Reply:
I corrected it in case of `ElasticCaseService`. In case ElasticTaskService the executor is not NAE class.

Reply:
We should probably use NAE class in task service as well - maybe create another task for this change